make/langtools/netbeans/nb-javac/build.xml

```diff
@@ -249,16 +239,29 @@
 
     <!-- suppress javadoc generation for this library, as its wrappers do it -->
     <target name="-javadoc-build"/>
-
+    
     <target name="-separate-jars" depends="init">
+        <copy  file="${build.classes.dir}/javax/lang/model/SourceVersion.class"
+               tofile="${build.classes.dir}/META-INF/versions/11/javax/lang/model/SourceVersion.class"
+               encoding="ISO-8859-1"
+               outputencoding="ISO-8859-1"
+        >
+            <filterchain>
+                 <replacestring from="nbjavac/RuntimeWR$Version" to="java/lang/Runtime$Version"/>
```

Comment:
The important trick is that both `nbjavac/RuntimeWR$Version` and `java/lang/Runtime$Version` have the same length.